migrate/20231222_minio_scale_out.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    alter_table(:minio_pool) do
+      add_column :server_count, :integer, null: true
+      add_column :drive_count, :integer, null: true
+      add_column :storage_size, :integer, null: true
```

Comment:
We have `storage_size_gib` examples in the codebase. Should we use it there too?